src/test/components/LocalTrack.test.js

```diff
@@ -0,0 +1,127 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+
+import {
+  changeSelectedThread,
+  hideLocalTrack,
+} from '../../actions/profile-view';
+import LocalTrack from '../../components/timeline/LocalTrack';
+import {
+  getLocalTracks,
+  getRightClickedTrack,
+} from '../../reducers/profile-view';
+import { getSelectedThreadIndex } from '../../reducers/url-state';
+import { getProfileWithNiceTracks } from '../fixtures/profiles/tracks';
+import { storeWithProfile } from '../fixtures/stores';
+
+const LEFT_CLICK = 0;
```

Comment:
seeing this (and since you added it also in the GlobalTrack test file, I wonder if these could be moved into `src/test/fixtures/utils.js`